toolkits/springer/packages/springer-hero/scss/10-settings/springer-hero.scss

```diff
@@ -0,0 +1,3 @@
+$hero--padding: spacing(24);
```

Comment:
you don't seem to be using this? Should it be used on line 32 of the springer-hero.scss file (and the value changed accordingly).

Reply:
Well spotted.